lib/applicationinsights.json

```diff
@@ -0,0 +1,6 @@
+{
+  "connectionString": "${file:/mnt/secrets/ccd-data-store/app-insights-connection-string}",
+  "role": {
+    "name": "CCD-DataStore"
+  }
```

Comment:
could add health sampling at the same time:
https://github.com/hmcts/spring-boot-template/blob/74e6e173e44ee952d5b7609192d90c999db2ba61/lib/applicationinsights.json#L6-L22